.github/workflows/desktop-build.yml

```diff
@@ -215,21 +215,21 @@ jobs:
             do_tests: 1
             make_package: 1
 
-          - target: 11_Big_Sur
-            os: macos-11
-            xcode: 12.5.1
+          - target: 12_Monterey
+            os: macos-12
+            xcode: 13.3
             qt_version: homebrew
             type: Release
             do_tests: 1
             make_package: 1
 
-#          - target: 12_Monterey
-#            os: macos-12
-#            xcode: 13.3
-#            qt_version: homebrew
-#            type: Release
-#            do_tests: 1
-#            make_package: 1
+          - target: 13_Ventura
```

Comment:
is there a benefit to having a separate macos 13 build? does it perform differently than the forwards compatible macos 12 build?

Reply:
I believe 12 is forward compatible

Reply:
All macOS builds should be forward compatible.
Is the 10.15 release not working on macOS 13? It should, no?

We used to only have binaries for the oldest macOS version we were supporting. There were never issues afaik.
Releasing multiple `.dmg`'s started when 10.14? offered dark mode support and we added a second release just for that feature.
Since then we kept having and maintaining multiple macOS releases in parallel.

Wouldn't it be easier to only ensure the oldest macOS version we support is working and offering users a single release file only? CI can build and test the next newer already as well of course.